docs/_posts/examples/3400-01-31-data-driven-lines.html

```diff
@@ -0,0 +1,266 @@
+---
+layout: example
+category: example
+title: Style lines using data-driven styling and the identity property function.
+description: Using an identity line-color property function for a visualization
+tags:
+  - layers
+---
+
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v9',
+    center: [-122.48383155304096, 37.82882682974591],
+    zoom: 16
+});
+
+var red = '#F7455D';
+var blue = '#33C9EB';
+
+map.on('load', function () {
+	map.addSource('lines-source', {
+	  'type': 'geojson',
+	  'data': {
+			  'type': 'FeatureCollection',
+			  'features': [
+			    {
+			      'type': 'Feature',
+			      'properties': {
+			        'color': red,
```

Comment:
Thanks @lucaswoj! I added the `red` and `blue` variables here to show the output color is controllable (the colors are a little more muted than the 'red' and 'blue' string output). But if you think the string should be here can certainly change.